src/vip_client/classes/VipLauncher.py

```diff
@@ -1713,21 +1721,21 @@ def _invalid_chars_for_vip(cls, value) -> list:
     
     # Function to assert file existence in the input settings
     @classmethod
-    def _first_missing_file(cls, value, location: str) -> str: 
+    def _missing_file(cls, value, location: str) -> str: 
         """
-        Returns the path the first non-existent file in `value` (None by default).
+        Returns true if the file `value` does not exist at `location`.
         - `value` can contain a single file path or a list of paths.
         - `location` refers to the storage infrastructure (e.g., "vip") to feed in cls._exists().
         """
         # Case : list of files
         if isinstance(value, list):
-            for file in value : 
-                if cls._first_missing_file(value=file, location=location) is not None:
+            for file in value :
+                if not cls._exists(file, location=location):
                     return file
-            return None
-        # Case: single file
+        # Case : single file
         else:
-            return value if not cls._exists(path=value, location=location) else None 
+            if not cls._exists(value, location=location):
+                return value
```

Comment:
Not sure a about the method name change.
I understand that if several files are missing in a directory it will still only return the first one missing, not all of them ?